tests/acceptance/editor-undo-redo-test.js

```diff
@@ -99,6 +92,46 @@ test('undo/redo the insertion of multiple characters', (assert) => {
   });
 });
 
+test('adding and deleting characters break the undo group/run', (assert) => {
```

Comment:
Yes,  or in the `undo/redo the insertion of multiple characters`  test above it.

Making the timeout configurable is a good option although I don't see people changing it too often if we come up with a timeout that works well.